lib/galaxy/config/sample/file_sources_conf.yml.sample

```diff
@@ -12,6 +12,10 @@
   root: ${user.preferences['owncloud|root']}
   login: ${user.preferences['owncloud|username']}
   password: ${user.preferences['owncloud|password']}
+  # Set the following settings to avoid loading entire files into memory
+  # useful when dealing with big files
+  use_temp_files: true
+  temp_path: /your/temp/path
```

Comment:
Can we default this to `new_file_path` if it is not set ? Also worth targeting older releases, probably 23.1 ?

Reply:
Yes we were just discussing it in https://github.com/galaxyproject/galaxy/issues/17372#issuecomment-1916595499

I will make it the default if it is not set and target 23.1 :+1:

Reply:
I couldn´t find a clean way of getting the config to use `new_file_path` so I relayed on `tempfile.gettempdir()` as the default. Hope that is ok.

Reply:
sure, we do override the tempdir with new_file_path anyway.